.helm/ecamp3/deploy.sh

```
@@ -0,0 +1,32 @@
+#!/bin/sh
+
+set -e
+
+SCRIPT_DIR=$(realpath "$(dirname "$0")")
+REPO_DIR=$(realpath "$SCRIPT_DIR/../..")
```

Shellcheck (0.10.0)

[warning] REPO_DIR appears unused. Verify use (or export if used externally).

(SC2034)